### ISSUE TITLE
[Website] Testing new sim pages

### ISSUE THREAD

**Issue by aaronsamuel137** [phetsims/tasks#217]
The new sim pages on phet-dev.colorado.edu need thorough testing. In addition, there have been changes to the front page, and to the log in page.

http://phet-dev.colorado.edu/

*Please note issues in the "website" repo, and use the "Sim Pages" label*
https://github.com/phetsims/website/issues

Please test the following across all of our supported browsers.
- Windows (IE, FIrefox, Chrome)
- Mac(Safari, Firefox, Chrome)
- iPad (Mobile Safari)

### New banner
- [x] Test links on the new banner for log-in, register, donate, html5 sims
- [x] Test tab highlight behavior in banner

### New login system
- [x] Make sure pages links that require authentication have the login popup come up and redirect to the requested page.
- [x] Test entering an incorrect login
- [x] Try logging in from different pages and links and see if the redirecting behavior seems correct

### New sim pages
Test the following in two locales besides English - one left to right and one right to left. Look for any formatting issues that come up. (Danish and Arabic are probably the best test cases)

- [x] Test all expandable text on new sim page
- [x] Test all links
- [x] Verify table sorting behavior is correct for activity table
- [x] Make sure video works properly when logged in and not when logged out (Right now the video is only up for area-builder)
- [x] Check that the urls look right
- [x] Testing switching between html and legacy sim pages
- [x] Make a note of any sponsors that extend beneath the bottom of the screenshot

Try and test anything that seems pertinent even if I forgot to mention it here. Let me know of anything that seems funky to you.

**Comment by ariel-phet** [comment]
@arnabp @bryo5363 @oliver-phet @phet-steele please make this your current top testing priority. This is our first pass of testing on this project so we want to flesh out as many bugs as possible. There will likely be another round of testing before we make everything live. Thanks!

**Comment by oliver-phet** [comment]
Can we set up a testing matrix for this? If we are testing across all supported browsers, I think it would easier to sign up for a test and record performance/bugs with a matrix.

**Comment by ariel-phet** [comment]
@oliver-phet - definitely, do you mind putting one together? (It only has to be browsers, devices shouldn't matter too much, except obviously mobile safari needs to be an ipad)

**Comment by ariel-phet** [comment]
@oliver-phet btw, I accidently put you with the "@" symbol instead of oliver o. - but great suggestion, and certainly won't hurt for all of the designers to be doing some dev testing as well.

**Comment by oliver-phet** [comment]
https://docs.google.com/spreadsheets/d/1Almjl_c4iYCigUUZLXu2_xwK48jOlfrPABxjsZ-4A6Y/edit?usp=sharing

**Comment by orejolaphet** [comment]
@ariel-phet @aaronsamuel137 more of just a design question, but it seems that all pages have the same width with the exception of the "About PhET" page and "For Teachers". These two pages are just substantially wider than all the others. Would it be more appropriate for all pages to have the same dimensions?

**Comment by oliver-phet** [comment]
@orejolaphet That is a known issue that we aren't addressing in this overhaul. But yes, we will be unifying things in the future.

**Comment by bryo5363** [comment]
@aaronsamuel137 I noticed some issues when trying to register on the site. Everything on the register site was working EXCEPT for Country selection. I continued to register and got the following error

![screen shot 2015-04-23 at 9 55 05 am](https://cloud.githubusercontent.com/assets/7717368/7301273/df8ef87a-e99e-11e4-98c8-9d1c2ba793ce.png)

**Comment by aaronsamuel137** [comment]
The register page is not finished - this is expected for now. I'll let you know when this page is ready for testing.

**Comment by ariel-phet** [comment]
I am currently going to close this issue. We can make a new task once the register page is finished and any other testing we need done.
